### PR TITLE
Update attachments field in preneed schema

### PR DESCRIPTION
**Pull request by jbalboni**
Adds a files array with the attachment id and using the field name from Lihan.

### CODE REVIEW

**Comment by lihanli** [comment]
there's an api for attachment_types so i assume they can change, maybe we should leave out the enum?

**Comment by jbalboni** [comment]
@lihanli Yeah, I want to ask them about that. Fetching this data from an api is sort of annoying.

File: dist/40-10007-schema.json [hunk 1]
```diff
@@ -596,11 +596,35 @@
...
+                  "1",
+                  "2",
+                  "3",
+                  "5",
+                  "6"
+                ]
+              }
+            },
+            "required": [
+              "attachmentId"
```
Comment (lihanli) [inline dist/40-10007-schema.json]: confirmationCode should be required too

**Comment by jbalboni** [comment]
@lihanli Updated.

**Pull request merged**